### PR TITLE
Changed workflow of export records.

### PR DESCRIPTION
**Pull request by tony-gaia**
Now option to lock records when exporting, and also to unlock filtered records. Both of these are only available to admins / data engineers.

Similarly admin / data engineer can lock / unlock records in the data management.

Did a bunch of minor style updates to buttons so that buttons do not have have a right margin of 10px by default, rather they need to use a class.

Other minor refactoring.

### CODE REVIEW

File: src/app/pages/data-view-export/view-records/view-records.component.ts [hunk 1]
```diff
@@ -186,30 +195,79 @@ export class ViewRecordsComponent implements OnInit {
...
+                    });
+                    this.isLocking = false;
+                },
+                () => {
+                    if (this.lockRecordsOnExport) {
+                        this.messageService.add({
+                            severity: 'success',
+                            summary: 'Export success',
+                            detail: `There records were successfully exported and have now been locked`
+                        });
```
Comment (serge-gaia) [inline src/app/pages/data-view-export/view-records/view-records.component.ts]: Don't we also want a message even when  `this.lockRecordsOnExport ` is `false` ? Maybe append `and have now been locked` in the message if `this.lockRecordsOnExport`

Comment (serge-gaia) [reply src/app/pages/data-view-export/view-records/view-records.component.ts]: Oh I didn't see we were in a outer `if (this.lockRecordsOnExport)`. So maybe the second `if (this.lockRecordsOnExport)` is not necessary.

File: src/app/pages/data-view-export/view-records/view-records.component.ts [hunk 2]
```diff
@@ -186,30 +195,79 @@ export class ViewRecordsComponent implements OnInit {
...
+        } else {
+            exportObservable.subscribe(
+                (blob: Blob) => saveBlob(blob),
+                (error: APIError) => {
+                    this.messageService.add({
+                        severity: 'error',
+                        summary: 'Export error',
+                        detail: `There were error(s) when exporting: ${error.msg}`
+                    });
+                }
```
Comment (serge-gaia) [inline src/app/pages/data-view-export/view-records/view-records.component.ts:248]: success message?

Comment (tony-gaia) [reply src/app/pages/data-view-export/view-records/view-records.component.ts:248]: Yes I thought about this, but the success message was the file downloading

File: src/app/pages/data-view-export/view-records/view-records.component.ts (hunk 1 above)
Comment (tony-gaia) [reply src/app/pages/data-view-export/view-records/view-records.component.ts]: Yes, I'll remove this inner if

Comment (tony-gaia) [reply src/app/pages/data-view-export/view-records/view-records.component.ts]: Removed

**Pull request merged**